util/fipstools/delocate/delocate.go

```diff
@@ -1834,6 +1841,13 @@ func transform(w stringWriter, inputs []inputFile) error {
 	}
 	w.WriteString(fmt.Sprintf(".file %d \"inserted_by_delocate.c\"%s\n", maxObservedFileNumber+1, fileTrailing))
 	w.WriteString(fmt.Sprintf(".loc %d 1 0\n", maxObservedFileNumber+1))
+	if d.processor == aarch64 {
+		// Grab the address of BORINGSSL_bcm_test_[start,end] via a relocation
+		// from a redirector function. For this to work, need to add the markers
+		// to the symbol table.
+		w.WriteString(fmt.Sprintf(".global BORINGSSL_bcm_text_start\n"))
+		w.WriteString(fmt.Sprintf(".type BORINGSSL_bcm_text_start, @function\n"))
```

Comment:
Just curious: can the symbol be added to they symbol table with another marker than `@function`?

Reply:
I'm not sure. I took inspiration from what the delocater script was doing with other functions.